src/psyclone/psyir/frontend/fortran.py

```diff
@@ -61,10 +61,11 @@ def __init__(self):
         self._processor = Fparser2Reader()
         SYMBOL_TABLES.clear()
 
-    def psyir_from_source(self, source_code):
+    def psyir_from_source(self, source_code, free_form = True):
```

Comment:
Please could you fix the formatting errors reported by pycodestyle for this file.